libraries/entities-renderer/src/RenderableModelEntityItem.cpp

```diff
@@ -162,9 +164,10 @@ void RenderableModelEntityItem::updateModelBounds() {
 
         // now recalculate the bounds and registration
         model->setScaleToFit(true, scaledDimensions);
-        model->setSnapModelToRegistrationPoint(true, registrationPoint);
+        model->setSnapModelToRegistrationPoint(!_useOriginalPivot, registrationPoint);
         updateRenderItems = true;
-        model->scaleToFit();
```

Comment:
I don't think we need to call scaleToFit here because we call simulate below which calls scaleToFit as necessary